robottelo/config/base.py

```diff
@@ -879,7 +879,7 @@ def _validate_robottelo_settings(self):
         """Validate Robottelo's general settings."""
         validation_errors = []
         browsers = ('selenium', 'docker', 'saucelabs')
-        webdrivers = ('chrome', 'firefox', 'ie', 'phantomjs', 'remote')
+        webdrivers = ('chrome', 'edge', 'firefox', 'ie', 'phantomjs', 'remote')
```

Comment:
Have you tried Windows 10? There's both Internet Explorer and Edge :) Don't ask why, it's Microsoft. So NACK for this suggestion. And another reason - what if i have Win7? Then i have IE only, no Edge, why shouldn't i be able to execute the tests in IE?